src/main/java/com/puppycrawl/tools/checkstyle/checks/UniquePropertiesCheck.java

```diff
@@ -104,9 +103,7 @@ protected void processFiltered(File file, FileText fileText) {
      *         file, 0 is returned
      */
     protected static int getLineNumber(List<String> lines, String keyName) {
```

Comment:
this method is not in use in Check so should be removed. Looks like it is required by Tests, Tests need to use new method.